### ISSUE TITLE
Additional arguments for summary.image

### ISSUE THREAD

**Issue by Garoe** [tensorflow/tensorflow#12883]
### System information
N/A 

### Describe the problem

It'd be nice to have more control over the color map applied to the image summaries. In particular having `vmin` and `vmax` arguments as in matplotlib [imshow](https://matplotlib.org/devdocs/api/_as_gen/matplotlib.pyplot.imshow.html) function.
If no value is given the default behaviour could be kept, but if specified, black would correspond to vmin and white to vmax.

### Source code / logs
N/A

**Comment by ppwwyyxx** [comment]
You can simply do:
```
img = (img - vmin) / (vmax - vmin) * 255
tf.summary.image('img', tf.cast(img, tf.uint8))
```
to get what you want.

**Comment by Garoe** [comment]
I wasn't asking for a workaround. I'm trying to make the point that it's common enough that it makes sense to have that code integrated in the summary.image() function as it has been done with the matplotlib equivalent.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by feihugis** [comment]
@alextp May I submit a PR for this feature?

**Comment by nfelt** [comment]
An update - after some discussion, our plans for the `tf.summary.image()` op are actually to keep it narrowly focused on the minimum necessary for summary export (i.e. PNG conversion), and thus not add further normalization controls.

In fact, for TF 2.0, we are planning on reducing the normalization provided by the op to just do [`tf.image.convert_image_dtype(tensor, tf.uint8, saturate=True)`](https://www.tensorflow.org/api_docs/python/tf/image/convert_image_dtype).  With that change, float tensors will always be scaled according to an expected range of `[0.0, 1.0)`, so it will no longer adjust the scaling range based on the input data max and min values.  This brings it into consistency with all the various `tf.image.*` ops which only support `tf.image.convert_image_dtype()` as an input conversion.

I understand that it adds some convenience to be able to use the op's built-in normalization for a wider range of numerical inputs, but ultimately it doesn't scale well to parameterize the op so that the normalization can be fine-tuned.  So for TF 2.0 we're doing only the most minimal conversion and recommending that users who need more than that do their own explicit normalization.
